opus/application/apps/ui/views.py

```diff
@@ -105,12 +107,31 @@ def api_last_blog_update(request):
             lastupdate = fp.read().strip()
     except:
         try:
-            log.error('api_last_blog_update: Failed to read file "%s"',
+            log.error('api_notifications: Failed to read file "%s"',
                       settings.OPUS_LAST_BLOG_UPDATE_FILE)
         except:
-            log.error('api_last_blog_update: Failed to read file UNKNOWN')
+            log.error('api_notifications: Failed to read file UNKNOWN')
```

Comment:
Change this line and the other "read file UNKNOWN" error to say whether it's a blog or notification file, since now there are two possibilities.